setup.py

```diff
@@ -365,8 +365,20 @@ def _classify_installed_files_override(
 
         print("Copying files from CMake output")
 
+        # lines for a proper work using pylint and an autocomplete in IDE
+        with open("%spython/cv2/__init__.py"
+        % cmake_install_dir, 'r') as opencv_init:
+            opencv_init_lines = opencv_init.readlines()
+            extra_imports = ('\nfrom .cv2 import *\nfrom . import data\n')
+            free_line_after_imports = 5
+            opencv_init_lines.insert(free_line_after_imports, extra_imports)
+
+        with open("%spython/cv2/__init__.py"
+        % cmake_install_dir, 'w') as opencv_final_init:
+            opencv_final_init.writelines(opencv_init_lines)
```

Comment:
The same logic as in https://github.com/opencv/opencv-python/pull/579, but we have here only two imports.

Reply:
os.path.join should be better than string substitution.

Reply:
Updated it in the latest commit